.github/workflows/release.yml

```diff
@@ -120,22 +120,19 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           DRY_RUN: true
 
-      - name: "3. Determine docker tags"
-        run: echo ::set-env name=DOCKER_TAGS::$(echo latest, ${{ steps.bump_dry.outputs.tag }})
+      - name: "3. Login to docker hub"
+        run: "echo '${{ secrets.DOCKER_HUB_TOKEN }}' | docker login --username ${{ secrets.DOCKER_HUB_USER }} --password-stdin"
```

Comment:
don't we have to pass the secret with env like here? https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#example-using-bash

Reply:
Can you do `docker login --username ${{ secrets.DOCKER_HUB_USER }} --password ${{ secrets.DOCKER_HUB_TOKEN }}"` ? In such case you will leave pasword in logs

Reply:
> don't we have to pass the secret with env like here? https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#example-using-bash

yes, probably you are right. I will change it soon

Reply:
> Can you do `docker login --username ${{ secrets.DOCKER_HUB_USER }} --password ${{ secrets.DOCKER_HUB_TOKEN }}"` ? In such case you will leave pasword in logs

According to [docker login](https://docs.docker.com/engine/reference/commandline/login/#login-to-a-self-hosted-registry) documentation using --password-stdin avoids logging credentials, but correct me if I misunderstood it.

Reply:
It is when you cat a file and then pipe it. In our case we echo it, so it will be visible. In a solution that I proposed it will as well because it inject it to command

Reply:
You need to use env - https://github.community/t/github-actions-docker-login/16053/2